schemas/com.olark/transcript/jsonschema/1-0-0

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an Olark transcript event",
+  "self": {
+    "vendor": "com.olark",
+    "name": "transcript",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "maxLength": 128
+      }
+    },
+    "items": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "body": {
+            "type": "string"
+          },
+          "timestamp": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "operatorId": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "kind": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "nickname": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "visitor_nickname": {
+            "type": "string",
+            "maxLength": 64
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "groups": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "kind": {
+            "type": "string",
+            "maxLength": 128
+          },
+          "name": {
+            "type": "string",
+            "maxLength": 128
+          },
+          "id": {
+            "type": "string",
+            "maxLength": 128
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "visitor": {
+      "type": "object",
+      "properties": {
+        "organization": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "city": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "ip": {
+          "type": "string",
+          "format": "ipv4"
+        },
+        "phoneNumber": {
+          "type": "string",
+          "maxLength": 64
+        },
+        "browser": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "fullName": {
+          "type": "string",
+          "maxLength": 256
+        },
+        "emailAddress": {
```

Comment:
```json
{
  "format": "email",
  "maxLength": 256
}
```

https://github.com/snowplow/iglu/pull/258

Reply:
@Chuwy I changed the format. About the field length I realise now I read the wrong RFC (RFC 3696), but I wonder if in the case of the ESP error in mailgun it can actually be 320?